midi/midifile.cpp

```diff
@@ -293,7 +293,10 @@ bool MidiFile::read(QIODevice* in)
                   break;
             default:
                   throw(QString("midi file format %1 not implemented").arg(_format));
+#if (!defined (_MSCVER) && !defined (_MSC_VER))
+                  // Prevent "unreachable code" warning in MSVC
```

Comment:
If that code is unreachable irrespectively of compiler used, remove it altogether